### ISSUE TITLE
Web Analog topic on Text is missing alignment details

### ISSUE THREAD

**Issue by mit-mit** [flutter/flutter#8760]
The Web Analog page has a section titled 'Styling and aligning Text':
https://flutter.io/web-analogs/#styling-and-aligning-text

However, it only seems to cover the styling part?

**Comment by mit-mit** [comment]
@LarkAscending, is this missing, or am I misunderstanding something?

**Comment by LarkAscending** [comment]
"In both HTML and Flutter, by default child elements or widgets are anchored at the top left."

It's intentional. This topic describes the default alignment, not how to re-align text. We could add something re:textAlign in the future, but it means changing every example.

**Comment by mit-mit** [comment]
Yes, but the code samples (both HTML and Flutter) do not shown any alignment changes, which is confusing when the heading says 'Styling and Aligning Text'.

Why not move the *"In both HTML and Flutter, by default child elements or widgets are anchored at the top left."* part down to the 'Centering Components' section; that matches the existing code sample there. Then we can change the 'Styling and Aligning Text' heading to just 'Styling Text', which again matches what the code describes.

I mention this not as an arbitrary review comment, but as a "user" of this page. I was trying to get alignment to work yesterday, and used this page as a reference. I got a bit confused when I could not find anything about how to change the alignment under the heading that had "aligning" in it.

**Comment by LarkAscending** [comment]
That's a good idea...except we also need to show aligning text. That's what
this is a placeholder for.
I'm working on this end to enhance that example that caused you issues.
(No, you're not supposed to know what the TW intends to do next.)

Aligning L/R and T/B and centering, they're all done differently.
So the centering example is not as related as it seems, but I think we can
enhance that first example and cascade the changes through the page.
WDYT?

Good user feedback, much appreciated!

**Comment by mit-mit** [comment]
> I think we can
enhance that first example and cascade the changes through the page.
WDYT?

Sounds great!
